llarp/service/info.hpp

```diff
@@ -66,18 +66,6 @@ namespace llarp
         return !(*this == other);
       }
 
-      ServiceInfo&
-      operator=(const ServiceInfo& other)
-      {
-        enckey  = other.enckey;
-        signkey = other.signkey;
-        version = other.version;
-        vanity  = other.vanity;
-        version = other.version;
-        UpdateAddr();
```

Comment:
`UpdateAddr` needs to be called on assignment to recompute the cached address

Reply:
Does it? Just naive copying should have the same effect?

Reply:
Just for citation, I think this is fine, because `UpdateAddr` just calls:

- `BEncode` into a temp array (which is stable)
- `crypto_generichash_blake2b` into `m_CachedAddr` (which is meant to be stable since it's a hash function)

this is replicable by just copying `m_CachedAddr`

Reply:
okay yeah that's fine